spotlight-client/src/SystemNarrativePage/SystemNarrativePage.tsx

```diff
@@ -148,14 +168,20 @@ const SystemNarrativePage: React.FC<{
 
   return (
     <Container>
-      <SectionNavigation
-        activeSection={activeSection}
-        // pagination UI should not respect the scrolling flag;
-        // in fact it should override it, otherwise the buttons
-        // will stop working while the animation is in progress
-        setActiveSection={directlySetActiveSection}
-        totalPages={narrative.sections.length + 1}
-      />
+      <NavContainer>
+        <Sticker>
+          <NavStickyContainer>
```

Comment:
I'm getting a console warning from the `NavStickyContainer`

<img width="1760" alt="Screen Shot 2021-01-12 at 11 17 36 AM" src="https://user-images.githubusercontent.com/5872651/104362074-2bcd4f80-54c8-11eb-9e86-a8170580720f.png">

Reply:
I also think that the mobile styling needs to be adjusted now that this NavStickyContainer has been added.

![Screen Shot 2021-01-12 at 11 19 24 AM](https://user-images.githubusercontent.com/5872651/104362115-3be52f00-54c8-11eb-9aa6-c5eaf9cc1bf3.png)

Reply:
oh yeah there is no mobile styling yet. designs should be landing sometime this week so I am punting it for now

Reply:
also, I saw that warning too. React is starting to complain about some of the dependencies using deprecated APIs. I am not super inclined to care about this at this point? It only gets raised in Strict Mode, which in practice I am finding pretty annoying and have been thinking about disabling. Maybe it's helpful as a debugging tool but as a general development tool I think it's way too noisy about stuff that's not in our direct control. (I have not been able to find a direct replacement for this dependency, one that simply wraps the polyfill in React, and I don't really think it's worth trying to figure out how to reimplement it without `findDOMNode`. There are various other pure-JS or pure-React libraries, in fact I used one of them in Spotlight v1 but the performance of this is way better in every browser except IE 11 because it uses the native CSS solution instead of janky JavaScript.)